contrib/corp_metr/info.yml

```diff
@@ -5,24 +5,24 @@ plot_name: 'Marginal Effective Total Tax Rates on Typically Financed Corporate I
 plot_id: corp_mettr
 content: index.html
 thumbnail: thumbnail.png
-short_description: > 
+short_description: >
     Explore the phaseout of bonus depreciation scheduled under current law.
 long_description: >
-    Explore the the impact of the scheduled phase-out of bonus depreciation on effective 
-    tax rates for new investment. Bonus depreciation will decline from 50% in 2016 and 
-    2017 to 40% in 2018, 30% in 2019, and will expire on December 31, 2019.  Calculations 
-    do not include the effect of tax reform on firms' financing behavior or the economy 
+    Explore the the impact of the scheduled phase-out of bonus depreciation on effective
+    tax rates for new investment. Bonus depreciation will decline from 50% in 2016 and
+    2017 to 40% in 2018, 30% in 2019, and will expire on December 31, 2019.  Calculations
+    do not include the effect of tax reform on firms' financing behavior or the economy
     as a whole.
 Concept_credit: >
     <a href = "http://jasondebacker.com/">Jason DeBacker</a>
 Development_credit: >
     <a href = "https://github.com/brendancol">Brendan Collins</a>
 OSS_credit: >
-    <a href = "https://github.com/open-source-economics/B-Tax">
-    Cost of Capital Calculator</a> for tax computations.
-    <a href = "http://bokeh.pydata.org/en/latest/">Bokeh</a>
+    <a href = "https://github.com/open-source-economics/B-Tax/releases/tag/0.1.2">
+    Cost of Capital Calculator version 0.1.2</a> for tax computations.
```

Comment:
@GoFroggyRun Do you think we credit B-Tax for this plot?

Reply:
Right, but I was thinking that we should credit B-Tax in the same way that we credit Tax-Calculator instead of TaxBrain for the other plots.

Reply:
Ahh. I see your point. I thought Coc and B-Tax are more like synonyms. But I am not sure now. It seems to me that we should credit B-Tax instead of COC.

Reply:
Yeah, @MattHJensen what do you think?